rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
```

Comment:
yes, just drop reporting if it contains any regex special character, that sounds good.
And you're welcome, it's been plenty useful to me :)